.circleci/config.yml

```diff
@@ -137,6 +137,25 @@ jobs:
       - *build_docker_image
       - *push_to_ecr
 
+  deploy_uat: &deploy_uat
+    executor: cloud-platform-executor
+    steps:
+      - checkout
+      - setup_remote_docker:
+          docker_layer_caching: true
+      - *decrypt_secrets
+      - *authenticate_k8s
+      - deploy:
+          name: Deploy LAA Estimate Eligibility Service to UAT
+          command: |
+            ./bin/uat_deployment 
+
+# this job does the same as deploy_uat however it will be
+# deploying to the main uat branch as it is only called
+# in the merge_pr_to_main workflow
+  deploy_main_uat:
```

Comment:
Also I don't understand why this is separate. Is it going to get its own steps? Its own parameters (not passed from the workflow)?

Reply:
the reason for this is really a circleci display thing. On the workflow pages e.g https://app.circleci.com/pipelines/github/ministryofjustice/laa-estimate-financial-eligibility-for-legal-aid/500/workflows/e72cf1a3-5e56-4630-a966-6777bcec3af3
the name given here is displayed in one of the boxes. This helps to make sense of the pipeline when viewing the visual workflow.

Reply:
Super explanation. In that case, I think [reusing the job directly but passing a `name` parameter](https://github.com/ministryofjustice/cla_backend/blob/master/.circleci/config.yml#L331-L332) may be best of both. What do you reckon?